proposals/NNNN-allows-navigation-apps-to-access-information-about-Wi-Fi-networks.md

```diff
@@ -18,7 +18,8 @@ Currently, there are some issues with navigation apps when users enable WiFi aft
 For example:
 1. SDL failed to start video streaming if users enable WiFi on the head unit after the NaviAPP is activated.
 https://github.com/smartdevicelink/sdl_java_suite/issues/1235
-2. SDL failed to start video streaming if users enable WiFi on the phone later 150s after the NaviAPP is activated.
+2. SDL failed to start video streaming if users enable WiFi on the phone more than 150s after the NaviAPP is activated.
+

```

Comment:
```suggestion
```

Reply:
@theresalech Thanks, fixed in [2abf8e3](https://github.com/smartdevicelink/sdl_evolution/pull/931/commits/2abf8e3edafa267afd0f372992d2bb48f3fb14b6)